docs/src/main/asciidoc/hibernate-reactive.adoc

```diff
@@ -357,27 +357,45 @@ public class SomeTest {
 ----
 ====
 
-[[hr-named-datasource]]
-==== Named data sources
+[[multiple-persistence-units]]
+=== Multiple persistence units
 
-Hibernate Reactive supports having named data sources
+==== Setting up multiple persistence units
+
+In a similar fashion to Hibernate ORM, Hibernate Reactive supports multiple persistence units.
+
+Users can define multiple persistence units and datasources, and they can mix blocking and reactive datasources.
+To ensure that a datasource supports reactive, you need to set the `reactive` property to `true`.
 
 [source,properties]
 .Example `{config-file}`
 ----
-# datasource configuration
-quarkus.hibernate-orm.datasource = named-datasource
-quarkus.datasource."named-datasource".db-kind" = postgresql
-
-# drop and create the database at startup (use `update` to only update the schema)
-%prod.quarkus.hibernate-orm.schema-management.strategy=drop-and-create
-%prod.quarkus.datasource."named-datasource".reactive" = true
-%prod.quarkus.datasource."named-datasource".username" = quarkus_test
-%prod.quarkus.datasource."named-datasource".password" = quarkus_test
-%prod.quarkus.datasource.reactive.url = vertx-reactive:postgresql://localhost/quarkus_test <1>
+quarkus.datasource."users".reactive=true <1>
+quarkus.datasource."users".reactive.url=vertx-reactive:postgresql://localhost/users
+quarkus.datasource."users".db-kind=postgresql
+quarkus.datasource."users".username=hibernate_orm_test<1> Define a datasource named `users`.
+quarkus.datasource."users".password=hibernate_orm_test
+
+quarkus.datasource."inventory".reactive=true <2>
+quarkus.datasource."inventory".reactive.url=vertx-reactive:postgresql://localhost/inventory
+quarkus.datasource."inventory".db-kind=postgresql
+quarkus.datasource."inventory".username=hibernate_orm_test
+quarkus.datasource."inventory".password=hibernate_orm_test
+
+quarkus.hibernate-orm."users".schema-management.strategy=drop-and-create <3>
```

Comment:
I believe you sent other PRs previously, at my request, to avoid this kind of configuration which could result in a _production database_ being dropped :)

Maybe see how it's done in quickstarts or the ORM documentation?